include/jemalloc/jemalloc_typedefs.h.in

```diff
@@ -75,3 +75,5 @@ struct extent_hooks_s {
 	extent_split_t		*split;
 	extent_merge_t		*merge;
 };
+
```

Comment:
nit: I noticed that this change was later reverted in the next commit; since this is a public header so I'm extra nitpicking -- let's remove it from this commit to avoid touching this file (this way the file history is not affected).

You can try `git rebase -i dev` to do an interactive rebase, which will allow you to edit commits in the middle, commit and then continue the rebase (and handle conflicts if any). 

In general when updating PR, we always try editing the exact commit directly, unless it would cause significant conflicts during the merge / rebase. For this PR I think it's already edited in the second commit, so I'd say only change this header file is fine.

Reply:
This is reasonable, thanks. Fixed.